### DISCUSSION TITLE
能不能加一个建筑公司停贷的项目

### DISCUSSION THREAD

**Discussion by Wesleylr** [WeNeedHome/SummaryOfLoanSuspension#871] [General]
rt，最近看到部分建筑公司也开始停贷了，建议加上去![image](https://user-images.githubusercontent.com/72133186/179524460-3c80ed34-44a6-4343-a083-28b2ffd625ce.jpeg)![image](https://user-images.githubusercontent.com/72133186/179524461-89f1297c-9173-4552-96de-a27b0c3c4655.jpeg)

**Comment by Wesleylr** [comment]
没人？

**Comment by blisee** [comment]
中华人民共和国的领导们出来看看人民的疾苦，给老百姓解决实际的问题，我们的钱到底去哪里了

**Reply by Wesleylr** [reply]
被他们自己手下吃了呗
